cacheops/redis.py

```diff
@@ -65,8 +66,25 @@ def __setattr__(self, name, value):
         self._setup()
         return setattr(self, name, value)
 
-redis_client = LazyRedis()
 
+#redis_client = LazyRedis()
+try:
+    CacheopsRedis = SafeRedis if settings.CACHEOPS_DEGRADE_ON_FAILURE else redis.StrictRedis
+    redis_replica_conf = settings.CACHEOPS_REDIS_REPLICA
+    redis_replica = redis.StrictRedis(**redis_replica_conf)
+
+    class ReplicaProxyRedis(CacheopsRedis):
+        """ Proxy `get` calls to redis replica.
+        """
+        def get(self, *args, **kwargs):
+            try:
+                return redis_replica.get(*args, **kwargs)
+            except redis.ConnectionError:
```

Comment:
the code has a handle_connection_failure already (in this file). it does not have retry though